pygeoapi/openapi.py

```diff
@@ -416,6 +426,831 @@ def get_oas_30(cfg):
             }
         }
 
+    # if CQL filter is applicable
+    if cql_filter_exists:
```

Comment:
if the configuration for filters is not set for any of the resources then **"/queryables"** (L431) wont be available(am still waiting for your decision on this) but **"collections/{collectionId}/queryables"** (L354) is available for all the resources irrespective of the CQL configuration(as it was before, I didn't apply any condition for it)

Reply:
@FarheenB Leave the higher level resource  "/queryables" (L431), I understand the logic but there are extra componentes (metdata records) that require it.  Leave it for now, this topic should be better checked.

@francbartoli   Trying to find some docs on this section "collections/{collectionId}/queryables" (L354)   even if we only have datetime, bbox,  we should have a list of queriables no ?

Reply:
sure @jorgejesus keeping this on hold for now

Reply:
@FarheenB 
- Based on the discussion in gitter the `/queriables` `collections/{collectionId}/queryables` have to be present, you need to update your code based on the disucussion

Reply:
Made the required changes for `/queryables` and `collections/{collectionId}/queryables`. please check https://github.com/FarheenB/pygeoapi/commit/f4aae51a804eb9913442f3fe6bf089ce7f397654 for document generation.
Since these two queryable endpoints are not specific to cql extension, so I removed its assertion from test_openapi.py. Please give a thought on this. @francbartoli @jorgejesus

Reply:
@FarheenB I would say the second `collections/{collectionId}/queryables` is actually relevant for CQL